src/test/runTest.sh

```diff
@@ -47,10 +52,43 @@ export LUA_INIT=''
 arg=''
 DEBUG_PREFIX=""
 
-echo "Running C tests"
-./histogram_test
-
 echo "Running: $ $LUA_BIN $LUA_ARGS -l histogram_c_test -e \"histogram_c_test.runTests()\""
 $LUA_BIN $LUA_ARGS -l histogram_c_test -e "histogram_c_test.runTests()"
 
+pushd ../python
+
+CIRC_PYTHON=/opt/circonus/python27/bin/python
+if [ -z "$PYTHON_BIN" ]
+then
+    if [ -x "$CIRC_PYTHON" ]
+    then
+        PYTHON_BIN=$CIRC_PYTHON
+    else
+        PYTHON_BIN="$(command -v python)"
+    fi
+fi
+
+if [ -x "$PYTHON_BIN" ]
+then
+    echo "Using Python interpreter $PYTHON_BIN"
+    $PYTHON_BIN --version
+else
+    echo "Python executable not found: $PYTHON_BIN"
+    exit 1
+fi
+
+if ! $PYTHON_BIN -c "import cffi" > /dev/null 2>&1;
+then
+    echo "Installing Python CFFI"
+    if ! $PYTHON_BIN setup.py develop --user > /dev/null 2>&1;
+    then
+        $PYTHON_BIN -m pip install cffi > /dev/null
```

Comment:
I would not want `make tests` to install anything on my system. It can provide instructions on the command line which I may or may not follow. Just installing stuff is a little rude.

Reply:
Agreed, but the install is not global. Notice no `sudo`.

Reply:
I still don't like it...
Installing dependencies should at least be a separate make target.